docs/src/pages/style/spacing.js

```diff
@@ -1,13 +1,52 @@
 import React from "react";
+import PropTypes from "prop-types";
 import { Helmet } from "react-helmet";
 import {
-  Box, Text, Title, SectionTitle, ListItem,
+  Box, Flex, Text, Title, SectionTitle, ListItem,
 } from "@nulogy/components";
 import {
   Layout, Intro, DocSection,
 } from "../../components";
+import theme from "../../../../components/src/theme";
 import anatomy from "../../images/spacing-anatomy.png";
-import scale from "../../images/spacing-scale.svg";
+
+const SpacingExample = props => {
+  const {
+    size,
+    scale,
+  } = props;
+  return (
+    <Box>
+      <Flex height="64px" verticalAlign="bottom" alignItems="flex-end">
+        <Box bg="lightBlue" mx="auto" mb="x2" height={ size } width={ size } />
+      </Flex>
+      <Text textAlign="center" mb="half">{ size }</Text>
+      <Text textAlign="center" fontSize="small">{ scale }</Text>
+    </Box>
+  );
+};
+
+SpacingExample.propTypes = {
+  size: PropTypes.string.isRequired,
+  scale: PropTypes.string.isRequired,
+};
+
+const SpacingExamples = props => {
+  const {
+    values,
+  } = props;
+
+  return (
+    Object.keys(values).map(key => {
```

Comment:
I remembered that there's a better way to do this.

Instead of the guard clause, you can filter for the stuff you want.

```
Object.keys(values)
  .filter(v => v.key !== "none")
  .map(key => <SpacingExample size={ values[key] } scale={ key } />);
```